Build fails on i386: error[E0412]: cannot find type `u512` in this scope

## Summary

```
error[E0412]: cannot find type `u512` in this scope
     --> /wrkdirs/usr/ports/lang/rustpython/work/target/release/build/unicode_names2-fc074b82fe4b3d20/out/generated.rs:13939:43
      |
13939 | pub static PHRASEBOOK_OFFSETS2: &'static [u512] = &[
      |                                           ^^^^ not found in this scope

```

unicode_names2's last release is 1.2.2, but some other tag  is used, which might be a cause of this problem.

[log](https://pkg-status.freebsd.org/beefy15/data/132i386-default/fbd796535d84/logs/rustpython-0.3.1.log)

rust-1.77.0
FreeBSD 13.2

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -79,7 +79,7 @@ static_assertions = "1.1"
 syn = "1.0.109"
 thiserror = "1.0"
 thread_local = "1.1.4"
-unicode_names2 = "1.1.0"
+unicode_names2 = "1.2.0"
 widestring = "1.1.0"
 windows-sys = "0.52.0"
 

